Thanks @jakobjakobson13 , the one CI failure looks real:

```
=================================== FAILURES ===================================
________________________ [doctest] scipy.constants.find ________________________
2039      'Stefan-Boltzmann constant']
2040 
2041     Get the constant called 'Boltzmann constant in Hz/K':
2042 
2043     >>> physical_constants['Boltzmann constant in Hz/K']
2044     (20836619120.0, 'Hz K^-1', 0.0)
2045 
2046     Find constants with 'radius' in the key:
2047 
2048     >>> find('radius')
Expected:
    ['Bohr radius',
     'classical electron radius',
     'deuteron rms charge radius',
     'proton rms charge radius']
Got:
    ['Bohr radius', 'alpha particle rms charge radius', 'classical electron radius', 'deuteron rms charge radius', 'proton rms charge radius']
```

--- a/scipy/constants/_codata.py
+++ b/scipy/constants/_codata.py
@@ -2047,6 +2047,7 @@ def find(sub: str | None = None, disp: bool = False) -> Any:
 
     >>> find('radius')
     ['Bohr radius',
+     'alpha particle rms charge radius',
      'classical electron radius',
      'deuteron rms charge radius',
      'proton rms charge radius']

